packages/neo-one-node-rpc/src/middleware/rpc.js

```diff
@@ -374,6 +374,13 @@ export default ({
         issueGASFee: issueGASFee.toString(),
       };
     },
+    runconsensusnow: () => {
+      if (node.consensus) {
+        node.consensus.runConsensusNow();
+      } else {
+        throw new Error('Something went wrong. consensus is empty');
```

Comment:
Make this error say `'This node does not support trigger consensus.'`. Also add a check here that this is a private network. If neo-one were ever to be used for mainnet or testnet consensus, we wouldn't want to expose a method to allow people to arbitrarily run consensus.